clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -1893,7 +1893,9 @@ class CIRGlobalOpLowering
     SmallVector<mlir::NamedAttribute> attributes;
     auto newGlobalOp = rewriter.replaceOpWithNewOp<mlir::LLVM::GlobalOp>(
         op, llvmType, op.getConstant(), convertLinkage(op.getLinkage()),
-        op.getSymName(), nullptr, /*alignment*/ 0,
+        op.getSymName(), nullptr,
+        /*alignment*/ op.getAlignment().has_value() ? op.getAlignment().value()
```

Comment:
`mlir::LLVM::GlobalOp` defines alignment as `OptionalAttr<I64Attr>:$alignment`, which means you can just pass `op.getAlignment()` directly.

Reply:
compile error if just using op.getAlignment(), so I'm using op.getAlignment().value_or(0).